swarm/pot/pot.go

```diff
@@ -477,18 +477,14 @@ func (t *Pot) each(f func(Val, int) bool) bool {
 	return f(t.pin, t.po)
 }
 
-// EachFrom called with (f, start) is a synchronous iterator over the elements of a Pot
+// eachFrom called with (f, start) is a synchronous iterator over the elements of a Pot
 // within the inclusive range starting from proximity order start
 // the function argument is passed the value and the proximity order wrt the root pin
 // it does NOT include the pinned item of the root
 // respecting an ordering
 // proximity > pinnedness
 // the iteration ends if the function return false or there are no more elements
 // end of a po range can be implemented since po is passed to the function
-func (t *Pot) EachFrom(f func(Val, int) bool, po int) bool {
```

Comment:
If this goes away then we should remove the non-exported `eachFrom()` as well - I wonder though if this function has some value to be kept.... @zelig?

Reply:
@holisticode the non-exported `eachFrom` is actually used. `git` search and you will find it.